packages/sonataflow-quarkus-devui/pom.xml

```diff
@@ -33,6 +33,7 @@
 
   <groupId>org.apache.kie.sonataflow</groupId>
   <artifactId>sonataflow-quarkus-devui-parent</artifactId>
+  <version>${revision}</version>
```

Comment:
please verify that when running `mvn clean install` from root works when you add this

Reply:
We have the GHA Build Midstream KIE Tools:
https://github.com/kubesmarts/kie-tools/actions/runs/17579758613
let see